pkg/util/flags/flags.go

```diff
@@ -82,24 +82,35 @@ func createRequiredFlagError(name string, flags []string) error {
 	return fmt.Errorf("%s are required for %s", strings.Join(flags, ","), name)
 }
 
-func SetLoggingFlags(flags *pflag.FlagSet) {
+func SetLoggingFlags(flags *pflag.FlagSet) error {
 	//Add commandline flags to the flagset. We will always write to stderr
 	//and not to a file by default
 	flags.AddGoFlagSet(flag.CommandLine)
-	flags.Set("logtostderr", "true")
-	flags.Parse(nil)
+	if err := flags.Set("logtostderr", "true"); err != nil {
+		return fmt.Errorf("error while setting the flag - %+v", err)
+	}
+	if err := flags.Parse(nil); err != nil {
+		return fmt.Errorf("error while parsing the flag -%+v", err)
+	}
+	return nil
 }
 
 func SetFlagsFromEnv(flags *pflag.FlagSet, prefix string) error {
+	errorFlag := false
+	var err error
```

Comment:
You don't need to declare it, simply use `:=` below.

Reply:
flags.VisitAll(.....) is a function and inside that function this `errorFlag` is being set to true. This function doesn't support a return call. That's why I initialized it at the beginning. The `errorFlag != false` is being checkout outside of the `VisitAll()` function. @leseb

Reply:
I can make it 
```var (
       errorFlag bool
       err error
  )
```

Reply:
I think Seb is referring to `var err error`

Reply:
Ah yes. That makes sense.Sure. I'll do the changes.

Reply:
Oh. I cannot make that change also. Because the error is also displayed outside this `VisitAll()` function.